styles/app.scss

```diff
@@ -4,3 +4,230 @@
 // $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 // @import 'font-awesome';
 @import 'box-sizing';
+@import 'variables';
+@import 'sidebar';
+
+body {
+  margin: 0;
+  width: 100vw;
```

Comment:
You can remove this width. It is automatically assumed that the body should be 100% of the viewport.